templates/pytorch-1.1.x/src/__ALGO__.py

```diff
@@ -32,8 +32,8 @@ def __init__(self, input_dict):
         else:
             raise Exception('input must be a json object.')
         if self.A.shape[-1] != self.B.shape[0]:
-            raise Exception('inner dimensions between A and B must be the same.\n A: {} B: {}'.format(self.A.shape[-1],
-                                                                                                      self.B.shape[0]))
+            raise Exception('inner dimensions between A and B must be the same.\n A: {} B: {}'\
+                .format(self.A.shape[-1], self.B.shape[0]))
```

Comment:
@jamesatha does this look better?

Reply:
I'm not a fan of the slash. I'd move the method to the previous line and have the args on the second line. But don't feel too strongly